test/e2e/node/node_problem_detector.go

```diff
@@ -233,3 +256,39 @@ func getCpuStat(f *framework.Framework, host string) (usage, uptime float64) {
 	usage *= 1e-9
 	return
 }
+
+func getNpdPodStat(f *framework.Framework, host string) (cpuUsage, rss, workingSet float64) {
+	summaryCmd := "curl localhost:10255/stats/summary"
+	result, err := framework.SSH(summaryCmd, host, framework.TestContext.Provider)
+	framework.ExpectNoError(err)
+	Expect(result.Code).To(BeZero())
+
+	var resultMap map[string]interface{}
```

Comment:
I think you can use https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/apis/stats/v1alpha1/types.go

Reply:
Done